api/src/services/authentication.ts

```diff
@@ -139,6 +139,19 @@ export class AuthenticationService {
 					await this.knex('directus_users').update({ status: 'suspended' }).where({ id: user.id });
 					user.status = 'suspended';
 
+					if (this.accountability) {
+						await this.activityService.createOne({
```

Comment:
As it is now a generic `UPDATE` action we should add an accompanying revisions record so auditor can see the data changed. [See sample usage](https://github.com/directus/directus/blob/5bdf3e6ed154d3e5d6f20db92a84b130cd529c40/api/src/services/items.ts#L346)